unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -689,6 +689,8 @@ mod tests {
     #[test_case("user_id in [1, 3, 6]", true)]
     #[test_case("user_id in [1, 3, 5]", false)]
     #[test_case("user_id not_in [1, 3, 5]", true)]
+    #[test_case("user_id in [\"dfsfsd\"]", false)]
```

Comment:
The grammar supports this case out the box but it's good to layer a test over this so we know for future